apps/server/src/modules/task/controller/api-test/task.api.spec.ts

```diff
@@ -1456,4 +1459,209 @@ describe('Task Controller (API)', () => {
 				.expect(501);
 		});
 	});
+
+	describe('Find tasks with assigned users', () => {
+		let app: INestApplication;
+		let em: EntityManager;
+		let apiRequest: TestRequest;
+
+		const setup = async () => {
```

Comment:
Can you please create a explict setup in each describe with when that only include the needed test data for this senario, instead of a "super setup" that include data for all senarios.

Reply:
This setup is almost as minimal as possible.
The edge cases can't be covered otherwise.

E.g. We can't check for finished tasks if we don't also have unfinished tasks in the db.
The tests could be restructured to contain much less data but that means we are not testing many known unknowns.